iib/workers/tasks/build_merge_index_image.py

```diff
@@ -118,16 +125,26 @@ def _add_bundles_missing_in_source(
             '%s bundles have invalid version label and will be deprecated.', len(invalid_bundles)
         )
 
-    _opm_index_add(
-        base_dir,
-        missing_bundle_paths,
-        binary_image,
-        from_index=source_from_index,
-        overwrite_from_index_token=overwrite_target_index_token,
-        # Use podman until opm's default mechanism is more resilient:
-        #   https://bugzilla.redhat.com/show_bug.cgi?id=1937097
-        container_tool='podman',
-    )
+    is_fbc = is_image_fbc(source_from_index)
```

Comment:
Can we rename `source_fbc` if we check the source image? Not important

Reply:
done